src/patternfly/components/Wizard/wizard.scss

```diff
@@ -0,0 +1,398 @@
+  --pf-c-wizard__main--PaddingLeft: var(--pf-global--spacer--xl);
+
+  // Footer
+  --pf-c-wizard__footer--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-wizard__footer--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-wizard__footer--PaddingBottom: var(--pf-global--spacer--lg);
+  --pf-c-wizard__footer--PaddingLeft: var(--pf-global--spacer--xl);
+
+  position: relative;
+  display: grid;
+  grid-template-rows: var(--pf-c-wizard--GridTemplateRows);
+  width: var(--pf-c-wizard--Width);
+  height: var(--pf-c-wizard--Height);
+  box-shadow: var(--pf-c-wizard--BoxShadow);
+
+  @media screen and (min-width: $pf-global--breakpoint--md) {
+    grid-template-columns: var(--pf-c-wizard--md--GridTemplateColumns);
+  }
+
+  > *:not(.pf-c-wizard__header),
+  .pf-c-wizard__nav {
+    background-color: var(--pf-c-wizard--children--BackgroundColor);
+  }
+
+  .pf-c-background-image {
+    &::before {
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+    }
+  }
+}
+
+.pf-c-wizard__header {
+  @include pf-t-dark;
+
+  position: relative;
+  padding: var(--pf-c-wizard__header--PaddingTop) var(--pf-c-wizard__header--PaddingRight) var(--pf-c-wizard__header--PaddingBottom) var(--pf-c-wizard__header--PaddingLeft);
+  grid-row: 1 / 2;
+
+  @media screen and (min-width: $pf-global--breakpoint--md) {
+    grid-column: 1 / -1;
+  }
+
+  // Nested selector to match button component color specificity
+  .pf-c-wizard__close {
+    position: absolute;
+    top: var(--pf-c-wizard__close--Top);
+    right: var(--pf-c-wizard__close--Right);
+    font-size: var(--pf-c-wizard__close--FontSize);
+  }
+}
+
+.pf-c-wizard__title {
+  padding-right: var(--pf-c-wizard__title--PaddingRight);
+
+  @media screen and (min-width: $pf-global--breakpoint--md) {
+    grid-column: 1 / 2;
+    grid-row: 1 / 2;
+  }
+}
+
+.pf-c-wizard__description {
+  display: none;
+  padding-top: var(--pf-c-wizard__description--PaddingTop);
+  color: var(--pf-c-wizard__description--Color);
+
+  @media screen and (min-width: $pf-global--breakpoint--md) {
+    display: block;
+  }
+}
+
+.pf-c-wizard__steps {
+  position: relative;
+  grid-row: 2 / 3;
+  min-width: 0;
+
+  @media screen and (min-width: $pf-global--breakpoint--md) {
+    overflow: auto;
+    -webkit-overflow-scrolling: touch;
+    grid-column: 1 / 2;
+    grid-row: 2 / 4;
+    box-shadow: var(--pf-c-wizard__steps--md--BoxShadow);
+  }
+}
+
+.pf-c-wizard__toggle {
+  position: relative;
+  z-index: var(--pf-c-wizard__toggle--ZIndex);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  padding: var(--pf-c-wizard__toggle--PaddingTop) var(--pf-c-wizard__toggle--PaddingRight) var(--pf-c-wizard__toggle--PaddingBottom) var(--pf-c-wizard__toggle--PaddingLeft);
+  border: 0;
+  box-shadow: var(--pf-c-wizard__toggle--BoxShadow);
+
+  &.pf-m-expanded {
+    .pf-c-wizard__toggle-icon {
+      transform: rotate(180deg);
+    }
+
+    + .pf-c-wizard__nav {
```

Comment:
It might be worth checking with a react dev on this, but using the `+` in this selector makes it dependent on HTML structure, and it would be ideal to have styles applied that are not dependent on HTML structure. 

For example, something I'd like to test for components like this menu is if we can implement them in a way that is similar to how we handle popovers (wrt handling and trapping focus). Currently, the Popover displays in a separate section of the DOM from the toggle

Reply:
Yep, good call! Updated the toggle/menu expanded styles to respond to a class on their parent element instead that can be toggled in react.